include/openmc/cell.h

```diff
@@ -23,17 +23,21 @@ namespace openmc {
 // Constants
 //==============================================================================
 
-// TODO: Convert to enum
-constexpr int FILL_MATERIAL {1};
-constexpr int FILL_UNIVERSE {2};
-constexpr int FILL_LATTICE {3};
-
-// TODO: Convert to enum
-constexpr int32_t OP_LEFT_PAREN   {std::numeric_limits<int32_t>::max()};
-constexpr int32_t OP_RIGHT_PAREN  {std::numeric_limits<int32_t>::max() - 1};
-constexpr int32_t OP_COMPLEMENT   {std::numeric_limits<int32_t>::max() - 2};
-constexpr int32_t OP_INTERSECTION {std::numeric_limits<int32_t>::max() - 3};
-constexpr int32_t OP_UNION        {std::numeric_limits<int32_t>::max() - 4};
+enum CellFillType
```

Comment:
[Prefer `enum class` to plain `enum`](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class)

Reply:
Also, the prevailing style when defining a struct/class is to group the opening brace with the name:
```C++
enum class CellFillType {
  ...
```

Reply:
OK, both of these things are now included in the PR.